manifests/init.pp

```diff
@@ -3,6 +3,7 @@
   String  $tomcat_user          = 'tomcat',
   String  $tomcat_group         = 'tomcat',
   Boolean $manage_user          = true,
+  String  $tomcat_download_url  = 'https://archive.apache.org/dist/tomcat/tomcat-9/v9.0.62/bin/apache-tomcat-9.0.62.tar.gz',
```

Comment:
Will we be regularly updating this value for new versions of Tomcat? If not, I wonder if it makes sense to not set a default, thus preventing users for accidentally using an old version of Tomcat in the future.

Reply:
I want a default since you can run the puppet module "out of the box" without any hiera data and it will start up. I do not believe we will regular update this value.

Reply:
Is there any URL that Tomcat provides that points to the latest version?

Reply:
I do not believe so. Only seen hacks to grab it.